number_parser/parser.py

```diff
@@ -22,18 +21,18 @@ def __init__(self, language):
         if language not in SUPPORTED_LANGUAGES:
             raise ValueError(f'"{language}" is not a supported language')
         language_info = getattr(import_module('number_parser.data.' + language), 'info')
-        self.unit_numbers = _normalize_dict(language_info["UNIT_NUMBERS"])
-        self.direct_numbers = _normalize_dict(language_info["DIRECT_NUMBERS"])
-        self.tens = _normalize_dict(language_info["TENS"])
-        self.hundreds = _normalize_dict(language_info["HUNDREDS"])
-        self.big_powers_of_ten = _normalize_dict(language_info["BIG_POWERS_OF_TEN"])
+        self.unit_numbers = _normalize_dict(language_info["NUMBERS"]["UNIT_NUMBERS"])
```

Comment:
I was planning to now create more variables like ```self.unit_number_ordinal``` and so on. The logic part would also be updated first checking cardinal and then ordinal. 
It might be a bit messy with all these instance variables, will try to remove any duplicacy in code. Is there a better approach ?